manim/mobject/functions.py

```diff
@@ -119,12 +119,12 @@ def generate_points(self):
 
 
 class FunctionGraph(ParametricFunction):
-    def __init__(self, function, color=YELLOW, **kwargs):
-        self.x_min = -config["frame_x_radius"]
-        self.x_max = config["frame_x_radius"]
+    def __init__(self, function, x_min=None, x_max=None, **kwargs):
+        self.x_min = x_min or -config["frame_x_radius"]
+        self.x_max = x_max or config["frame_x_radius"]
```

Comment:
If `x_min` or `x_max` are `0`, then the values from the config will be used instead. Best to explicitly check `if x_min is None:` and such

Reply:
My bad, thanks for pointing out.